plugins/outputs/kafka/kafka.go

```diff
@@ -316,13 +318,14 @@ func (k *Kafka) Write(metrics []telegraf.Metric) error {
 	for _, metric := range metrics {
 		buf, err := k.serializer.Serialize(metric)
 		if err != nil {
-			log.Printf("D! [outputs.kafka] Could not serialize metric: %v", err)
+			k.Log.Debugf("Could not serialize metric: %v", err)
```

Comment:
I think it's more like error than debug information. Because it's about data loss, and not about "how and where?". It's my opinion.

Reply:
No change is made here to the level, so I think we can handle this in a separate issue.  I think it may have been added at some point due to metrics without fields, need to do some research on the history.